src/classes/CRLP_ApiService_TEST.cls

```diff
@@ -200,13 +198,14 @@ private class CRLP_ApiService_TEST {
      */
     @isTest
     private static void getBaseRollupStateForRecordsShouldBuildCRLPRollupMap() {
+        final String MOCK_ROLLUP_DEFINITIONS = '[{"mockRollupDefinition":"randomData"}]';
```

Comment:
@force2b turns out as long as this is valid JSON, it will still cast without breaking. I updated the variable to make it more clear that it is just mock data and not dependent on an actual serialized `List<CRLP_Rollup>`